DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Geometry/PointToSpeckleConverter.cs

```diff
@@ -17,13 +15,13 @@ public PointToSpeckleConverter(IConversionContextStack<Map, Unit> contextStack)
     _contextStack = contextStack;
   }
 
-  public Base Convert(object target) => RawConvert((ArcMapPoint)target);
+  public Base Convert(object target) => RawConvert((MapPoint)target);
```

Comment:
You are implementing the `Base Convert(object target)` method, but you aren't implementing the IHostObjectToSpeckleConversion interface. This isn't necessarily an issue; I'm just drawing your attention to it because it may be the result of a forgotten interface.

Reply:
Good catch, this can and should be removed to prevent confusion.

Reply:
At least at one point (Kat will confirm if this is still the case), this is an intentional omission because the `PointToSpeckleConverter` is only used by other conversions (via DI). We're not expecting MapPoints to be passed directly to top level converter, therefore we don't need this converter to match as a `IHostObjectToSpeckleConversion`